nsv13/code/modules/power/rbmk.dm

```diff
@@ -0,0 +1,864 @@
+			var/obj/structure/cable/C = T.get_cable_node()
+			if(!C || !C.powernet)
+				return
+			else
+				C.powernet.newavail += last_power_produced
+		var/total_control_moles = moderator_input.get_moles(/datum/gas/nitrogen) + (moderator_input.get_moles(/datum/gas/carbon_dioxide)*2) + (moderator_input.get_moles(/datum/gas/pluoxium)*3) //N2 helps you control the reaction at the cost of making it absolutely blast you with rads. Pluoxium has the same effect but without the rads!
+		if(total_control_moles >= minimum_coolant_level)
+			var/control_bonus = total_control_moles / 250 //1 mol of n2 -> 0.002 bonus control rod effectiveness, if you want a super controlled reaction, you'll have to sacrifice some power.
+			control_rod_effectiveness = initial(control_rod_effectiveness) + control_bonus
+			radioactivity_spice_multiplier += moderator_input.get_moles(/datum/gas/nitrogen) / 25 //An example setup of 50 moles of n2 (for dealing with spent fuel) leaves us with a radioactivity spice multiplier of 3.
+			radioactivity_spice_multiplier += moderator_input.get_moles(/datum/gas/carbon_dioxide) / 12.5
+		var/total_permeability_moles = moderator_input.get_moles(/datum/gas/bz) + (moderator_input.get_moles(/datum/gas/water_vapor)*2) + (moderator_input.get_moles(/datum/gas/hypernoblium)*10)
+		if(total_permeability_moles >= minimum_coolant_level)
+			var/permeability_bonus = total_permeability_moles / 500
+			gas_absorption_effectiveness = gas_absorption_constant + permeability_bonus
+		var/total_degradation_moles = moderator_input.get_moles(/datum/gas/nitryl) //Because it's quite hard to get.
+		if(total_degradation_moles >= minimum_coolant_level*0.5) //I'll be nice.
+			depletion_modifier += total_degradation_moles / 15 //Oops! All depletion. This causes your fuel rods to get SPICY.
+			playsound(src, pick('sound/machines/sm/accent/normal/1.ogg','sound/machines/sm/accent/normal/2.ogg','sound/machines/sm/accent/normal/3.ogg','sound/machines/sm/accent/normal/4.ogg','sound/machines/sm/accent/normal/5.ogg'), 100, TRUE)
+		//From this point onwards, we clear out the remaining gasses.
+		moderator_input.clear() //Woosh. And the soul is gone.
+		K += total_fuel_moles / 1000
+	var/fuel_power = 0 //So that you can't magically generate K with your control rods.
+	if(!has_fuel())  //Reactor must be fuelled and ready to go before we can heat it up boys.
+		K = 0
+	else
+		for(var/obj/item/fuel_rod/FR in fuel_rods)
+			K += FR.fuel_power
+			fuel_power += FR.fuel_power
+			FR.deplete(depletion_modifier)
+	//Firstly, find the difference between the two numbers.
+	var/difference = abs(K - desired_k)
+	//Then, hit as much of that goal with our cooling per tick as we possibly can.
+	difference = CLAMP(difference, 0, control_rod_effectiveness) //And we can't instantly zap the K to what we want, so let's zap as much of it as we can manage....
+	if(difference > fuel_power && desired_k > K)
```

Comment:
can you put the section before and after the && in their own () to make it easier to read please.